bin/cloud-local.sh

```diff
@@ -38,8 +38,7 @@ function download_packages {
   if [[ ! -z ${pkg_pre_download+x} ]]; then
     # Does that folder actually exist?
     if [[ -d ${pkg_pre_download} ]] ; then
-      test -d ${CLOUD_HOME}/pkg || rmdir ${CLOUD_HOME}/pkg
-      test -h ${CLOUD_HOME}/pkg && rm ${CLOUD_HOME}/pkg
+      rm -f pkg
```

Comment:
I can agree with the simplification but I don't always run commands from the $CLOUD_HOME directory. Should we make it `rm -f ${CLOUD_HOME}/pkg` to be sure?